Source/WebKit/UIProcess/mac/PageClientImplMac.mm

```diff
@@ -580,6 +580,11 @@ - (BOOL)_hostsLayersInWindowServer;
     m_impl->setTextIndicator(textIndicator.get(), lifetime);
 }
 
+void PageClientImpl::updateTextIndicator(Ref<TextIndicator> textIndicator)
```

Comment:
As Abrar mentioned in a previous comment, this (and nearby code) should either pass `Ref<TextIndicator>&&` or `const TextIndicator&`. Maybe we can clean that up separately, though.

Reply:
Yes, I said I’d take care of that separately, there’s going to be serval follow on patches to this problem, it could also go in one of those.